nextjs-app-v14/src/app/integrations/form/form.tsx

```diff
@@ -189,6 +196,17 @@ export default function Form() {
                     <IconKey slot="leadingIcon"></IconKey>
                 </PieTextInput>
 
+                <PieFormLabel for="description">
+                    Description:
+                </PieFormLabel>
+                <PieTextarea
+                    className="form-field"
+                    id="description"
+                    data-test-id="description"
+                    name="description"
+                    value={description}
+                    onInput={handleDescriptionTextarea as any}/>
```

Comment:
I'm curious if/why `as any` was needed. There seem to be lots of other functions in this file that don't need it 🤔